twentytwenty-blocks/style.css

```
@@ -0,0 +1,76 @@
+/*
+Theme Name: Twenty Twenty
```

```suggestion
Theme Name: Twenty Twenty Blocks
```

^ Updated to differentiate from the existing version of Twenty Twenty.